easybuild/easyconfigs/a/Amber/Amber-20.11-foss-2020a-AmberTools-20.15-Python-3.8.2.eb

```diff
@@ -0,0 +1,52 @@
+name = 'Amber'
+local_amber_ver = 20
+local_ambertools_ver = 20
+# Patch levels from http://ambermd.org/AmberPatches.php and http://ambermd.org/ATPatches.php
+patchlevels = (15, 11)  # (AmberTools, Amber)
+version = '%s.%s' % (local_amber_ver, patchlevels[1])
+versionsuffix = '-AmberTools-%s.%s-Python-%%(pyver)s' % (local_ambertools_ver, patchlevels[0])
+
+homepage = 'http://ambermd.org/amber.html'
+description = """Amber (originally Assisted Model Building with Energy Refinement) is software for performing
+ molecular dynamics and structure prediction."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+sources = [
+    '%%(name)s%s.tar.bz2' % local_amber_ver,
+    'AmberTools%s.tar.bz2' % local_ambertools_ver,
+]
+patches = [
+    '%%(name)s-%s_cmake-locate-netcdf.patch' % local_amber_ver,
```

Comment:
This is an AmberTools patch, not Amber, and should thus be named AmberTools-20_cmake... to make it easier to have a separate AmberTools package for those who don't want to buy Amber